src/sentry/static/sentry/app/views/settings/components/settingsBreadcrumb/index.jsx

```diff
@@ -21,22 +26,29 @@ const MENUS = {
 class SettingsBreadcrumb extends React.Component {
   static propTypes = {
     routes: PropTypes.array,
+    pathMap: PropTypes.object,
```

Comment:
Might help readers a bit to explain that `pathMap` comes from store and that it maps a raw route (w/o route params) -> title?

Reply:
2fc9ea6